From 95550e5a90d65b049dabcd31b7168612c9802327 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Florian=20M=C3=BCllner?= <fmuellner@gnome.org>
Date: Mon, 4 Nov 2013 20:09:45 +0100
Subject: [PATCH] network: Do not use timestamp to identify connections

Connections have UUIDs to uniquely identify connections, bringing
in timestamps just messes things up unnecessarily (and was dropped
upstream in 3.10).
Patch by Dan Williams.
---
 js/ui/status/network.js | 4 +---
 1 file changed, 1 insertion(+), 3 deletions(-)

diff --git a/js/ui/status/network.js b/js/ui/status/network.js
index bfd07fd..0a8871c 100644
--- a/js/ui/status/network.js
+++ b/js/ui/status/network.js
@@ -231,9 +231,7 @@ const NMConnectionBased = new Lang.Class({
         if (exists) {
             let existing = this._connections[pos];
 
-            // Check if connection changed name or id
-            similar = existing.name == connection.get_id() &&
-                existing.timestamp == connection._timestamp;
+            similar = existing.name == connection.get_id();
         }
 
         if (exists && valid && similar) {
-- 
1.8.4.2

